Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Compact/BasicOperators/CompactNegativeNodeable.cpp

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "CompactNegativeNodeable.h"
+
+namespace ScriptCanvas::Nodeables
+{
+    float CompactNegativeNodeable::In(float a)
+    {
+        return a * -1;
```

Comment:
I dont think you need to use '*' here
```suggestion
        return -a;
```